src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -437,19 +510,23 @@ export default class PhotoField extends React.Component {
             <ProgressBar percent={this.state.progress}/>
           </div>}
           {!isDone && this.state.src && <div className="cropper-container-outer">
-            <Cropper
-              ref="cropper"
-              ready={this.setCropBox}
-              responsive
-              src={this.state.src}
-              aspectRatio={1}
-              cropBoxMovable={false}
-              cropend={this.handleCropend}
-              toggleDragModeOnDblclick={false}
-              dragMode="move"
-              guides={false}
-              viewMode={1}
-              zoom={this.onZoom}/>
+            <div ref="container">
+              <Cropper
+                ref="cropper"
+                ready={this.setCropBox}
+                responsive
+                src={this.state.src}
+                aspectRatio={1}
+                cropBoxMovable={false}
+                cropend={this.handleCropend}
+                cropmove={this.handleCropMove}
+                minContainerHeight={smallScreen ? 240 : 300}
+                toggleDragModeOnDblclick={false}
+                dragMode="move"
+                guides={false}
+                viewMode={0}
```

Comment:
The old `viewMode` value was 1 which
> `restrict the crop box to not exceed the size of the canvas.`
The side effect is that the cropper size is adjusted to meet the default behavior of displaying the full image when it is initially loaded and the cropper size cannot be manually set. 

`viewMode={0}` allows the cropBox size to be manually set but does not constrain the canvas movement to the bounds of the cropBox necessitating handling this manually. 
<img width="588" alt="screen shot 2018-02-06 at 20 38 51" src="https://user-images.githubusercontent.com/4998130/35895812-bdcfa310-0b7e-11e8-8f37-4faa405d4e43.png">